client/src/ui/molecules/breadcrumbs/index.tsx

```diff
@@ -1,12 +1,15 @@
-import { DocParent } from "../../../document/types";
+import { Doc } from "../../../document/types";
 import { PreloadingDocumentLink } from "../../../document/preloading";
 
 import "./index.scss";
 
-export const Breadcrumbs = ({ parents }: { parents: DocParent[] }) => {
-  if (!parents.length) {
-    throw new Error("Empty parents array");
-  }
+export const Breadcrumbs = ({ doc }: { doc: Doc }) => {
+  const items = doc.parents || [
```

Comment:
You could type this as `DocParent[]` for safety.

Reply:
Fixed in 8e3010081.

I wonder this property on `Doc` shouldn't be called `breadcrumbs` or `path` instead of `parents`. It's weird to me that the `Doc` itself is its own parent.

Reply:
Also, if the `mdn_url` property was named `uri`, every `Doc` would implicitly be a `DocParent` (or, as I would tend to call it, `BreadcrumbItem` or `DocPathItem`).